package.json

```diff
@@ -74,7 +74,7 @@
     "jsonwebtoken": "^8.4.0",
     "lodash": "^4.17.11",
     "mime": "^2.4.0",
-    "minimed-connect-to-nightscout": "^1.1.1",
+    "minimed-connect-to-nightscout": "https://github.com/szymjaw/minimed-connect-to-nightscout.git#dev-servers",
```

Comment:
@szymjaw @mddub can you release an publish `minimed-connect-to-nightscout` to npmjs.org? We don't allow links to github branches from security perspective. Please make sure https://github.com/mddub/minimed-connect-to-nightscout/pull/11 is merged and the new version is published.